ui/diagnoseReportApp/translations/en.yaml

```
@@ -1,103 +1,103 @@
-      diagnose_in_time_range_t1: Diagnose in time range t1
-      diagnose_in_time_range_t2: Diagnose in time range t2
-      max_diff_item: Max Diff Item
-      slow_query_t2: Slow Query Only Appear In t2
-      generate_report_error: Generate Report Error
-      report_time_range: Report Time Range
-      diagnose: Diagnosis
-      total_time_consume: Total Time Consume
-      total_error: Total Error
-      time_consume: Time Consume
-      tidb_time_consume: TiDB time Consume
-      transaction: Transaction
-      tidb_connection_count: TiDB Connection count
+      compare_diagnose: Diagnostics Comparison
+      compare_report_time_range: Comparison Time Range
+      top_10_slow_query_in_time_range_t1: Top 10 Slow Queries in Time Range t1
+      top_10_slow_query_in_time_range_t2: top 10 Slow Queries in Time Range t2
+      top_10_slow_query_group_by_digest_in_time_range_t1: Top 10 Slow Queries Group by Digest in Time Range t1
+      top_10_slow_query_group_by_digest_in_time_range_t2: Top 10 slow queries group by digest in Time Range t2
+      slow_query_with_diff_plan in_in_time_range_t1: Slow Queries with Different Plan in Time Range t1
+      slow_query_with_diff_plan in_in_time_range_t2: Slow queries with Different Plan in Time Range t2
+      diagnose_in_time_range_t1: Diagnostics in Time Range t1
+      diagnose_in_time_range_t2: Diagnostics in Time Range t2
+      max_diff_item: Maximum Different Item
+      slow_query_t2: Slow Queries In Time Range t2
+      generate_report_error: Report Generation Error
+      report_time_range: Diagnostics Time Range
+      diagnose: Diagnosis Result
+      total_time_consume: Time Consumed by Each Component
+      total_error: Errors Occurred in Each Component
+      time_consume: Time Consumed
+      tidb_time_consume: Time Consumed by TiDB Component
+      transaction: TiDB Transaction
+      tidb_connection_count: TiDB Server Connections
       statistics_info: Statistics Info
       ddl_owner: DDL Owner
       scheduler_initial_config: Scheduler Initial Config
-      scheduler_change_config: Scheduler Change Config
+      scheduler_change_config: Scheduler Config Change History
       tidb_gc_initial_config: TiDB GC Initial Config
-      tidb_gc_change_config: TiDB GC Change Config
+      tidb_gc_change_config: TiDB GC Config Change History
       tikv_rocksdb_initial_config: TiKV RocksDB Initial Config
-      tikv_rocksdb_change_config: TiKV RocksDB Change Config
+      tikv_rocksdb_change_config: TiKV RocksDB Config Change History
       tikv_raftstore_initial_config: TiKV RaftStore Initial Config
-      tikv_raftstore_change_config: TiKV RaftStore Change Config
-      pd_time_consume: PD Time Consume
-      blance_leader_region: Blance Leader/Region
-      approximate_region_size: Approximate Region size
+      tikv_raftstore_change_config: TiKV RaftStore Config Change History
+      pd_time_consume: Time Consumed by PD Component
+      blance_leader_region: Scheduled Leader/Region Count
```

```suggestion
      blance_leader_region: Scheduled Leader/Region Count
```
typo: s/blance/balance?

---

@crazycs520

---

yes, should be balance

---

fix in https://github.com/pingcap-incubator/tidb-dashboard/pull/495